Detectors/ITSMFT/MFT/workflow/src/TrackerSpec.cxx

```diff
@@ -129,21 +129,21 @@ void TrackerDPL::run(ProcessingContext& pc)
         LOG(INFO) << "ROframe: " << roFrame << ", clusters loaded : " << nclUsed;
         mTracker->setROFrame(roFrame);
         mTracker->clustersToTracks(event);
-        tracks.swap(mTracker->getTracks());
+        //tracks.swap(mTracker->getTracks());
```

Comment:
Please do not comment code. Remove if not needed.

Reply:
Hi @ktf , I reviewed the code and removed some commented code. I am keeping those used for debugging.